### ISSUE TITLE
DDA and DIA libraries for DIA data analysis

### ISSUE THREAD

**Issue by cutleraging** [Nesvilab/FragPipe#1037]
Hello,

Thank you for your great work on fragpipe. I had run a pilot experiment which generated both DDA and DIA data from the same samples. Now I ran a larger experiment and generated only DIA data. I am now using the DIA_SpecLib_Quant workflow to analyze this data.

I am wondering if it makes sense to use both the DDA and DIA data from he pilot as libraries when analyzing the larger experiment? Or if I should just use the DDA data. The number of identified proteins and CV is similar between these. Is more relevant data for libraries always better?

Thanks,
Ronnie

**Comment by fcyu** [comment]
Hi Ronnie,

> I am wondering if it makes sense to use both the DDA and DIA data from he pilot as libraries when analyzing the larger experiment? Or if I should just use the DDA data. The number of identified proteins and CV is similar between these. Is more relevant data for libraries always better?

I think it might depend on your data. We tested DIA only vs DIA+DDA, in which DIA+DDA always gave more quantified IDs. But did not look deep into DDA only vs DIA+DDA. We will investigate it later.

Thanks for the feedback,

Fengchao

**Comment by anesvi** [comment]
There is also a benefit if suing DIA data as part of spectral library building in that fractionated DDA runs are all aligned in RT to a single shot DIA run (one run selected as a reference run for alignment). If only building the library from fractionated DDA data, need to use ciRT option (works for Human data only) for alignment in EassyPQP, or need to use iRT peptides. Processing DDA and DIA together simplifies that part

**Comment by cutleraging** [comment]
Thanks for the feedback. To be more specific about ho

**Comment by cutleraging** [comment]
Thanks for the feedback. So your recommendation would be to use the DDA and DIA data from the pilot experiment to build the spectral libraries? I did this by doing the following with the pilot experiment data: 1) including the DDA files and setting to DDA Data type 2) including the DIA files and setting Data type to DIA-lib. Then I included the DIA data from the larger experiment and set the data type to DIA (not DIA-Quant).

From my understanding this will create a spectral library from all of the files (DDA and DIA from the pilot, DIA from larger experiment), but only quantify the DIA samples from the larger experiment. Is this the right way to do this?

Thanks,
Ronnie
